libs/blocks/quiz-entry/quiz-entry.js

```diff
@@ -105,11 +105,11 @@ const App = ({
     }
   }, [countSelectedCards]);
 
-  let minSelections = 0;
+  // let minSelections = 0;
```

Comment:
Same with this, needed? Can we restore it?

Reply:
Not sure for this, but I just leave it there